### PR TITLE
Fix no output when Client=False,TestClient=True

### PR DESCRIPTION
**Pull request by severnt**
Up until 1.19.0, passing `Client=False` and `TestClient=True` would result in a file being generated. After commit https://github.com/grpc/grpc-swift/commit/09c46b0a6582652ebddb9bd6967ca2eb8fc67073 - this no longer happens.

### CODE REVIEW

**Review by glbrntt — APPROVED** [review]
Thank you!

File: Sources/protoc-gen-grpc-swift/main.swift [hunk 1]
```diff
@@ -156,7 +156,7 @@ func main(args: [String]) throws {
         binaryFile.content = serializedFileDescriptorProto
         response.file.append(binaryFile)
       }
-      if !fileDescriptor.services.isEmpty && (options.generateClient || options.generateServer) {
+      if !fileDescriptor.services.isEmpty && (options.generateClient || options.generateServer || options.generateTestClient) {
```
Comment (glbrntt) [inline Sources/protoc-gen-grpc-swift/main.swift:159]: Oh sorry, the formatter doesn't like this, could you run `./scripts/format.sh -f`?

**Review by glbrntt — CHANGES_REQUESTED** [review]

File: Sources/protoc-gen-grpc-swift/main.swift (hunk 1 above)
Comment (severnt) [reply Sources/protoc-gen-grpc-swift/main.swift]: Looks like the formatter script isn't happy with `-f`.. I'll put up a separate PR for that.

Comment (glbrntt) [reply Sources/protoc-gen-grpc-swift/main.swift]: Ah, right, `-f` isn't required but the script tells you to pass it 🙈

**Review by glbrntt — APPROVED** [review]

**Pull request merged**